Loop/Managers/DeviceDataManager.swift

```diff
@@ -697,8 +789,9 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
             if let sentrySupported = pumpState?.pumpModel?.larger where !sentrySupported {
                 rileyLinkManager.idleListeningEnabled = false
             }
-
             NSUserDefaults.standardUserDefaults().pumpModelNumber = pumpState?.pumpModel?.rawValue
+            self.enableRileyLinkHeartbeatIfNeeded()
```

Comment:
The G5 transmitter and G4 receiver are the only other reliable heartbeats, and they're preferred due to their longer (5 min) interval